src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Todo.css';
+
+export const Todo = ({ title, completed, user }) => (
+  <div className="Todo">
+    <h2 className="Todo__title">
+      {title}
+    </h2>
+    <p className="Todo__user">{user.name}</p>
+    {completed
+      ? (
+        <span
+          className="Todo__status Todo__status--done"
```

Comment:
You could create only one `span` instead of two with text `{completed ? 'Completed' : 'Not completed'}` and use `classnames` library